src/kayenta/edit/metricList.tsx

```diff
@@ -148,6 +156,14 @@ function mapDispatchToProps(dispatch: (action: Action & any) => void): IMetricLi
         }
       }));
     },
+    copyMetric: (metric: ICanaryMetricConfig) => {
+      const metricCopy: ICanaryMetricConfig = Object.assign({}, metric, {
+        id: '[new]',
```

Comment:
I think `id` needs to be unique across metrics (at least to avoid that React "duplicate key in list error"). What happens if you clone more than one metric?

Reply:
I think when a metric is added, it's assigned a new id by the `idMetrics` function in `selectedConfig.ts`. I copied the `[new]` temp ID from what's happening in `addMetric` but let me know if we should handle differently when copying.